database/seedRecords.js

```diff
@@ -0,0 +1,94 @@
+const faker = require('faker');
+const fs = require('fs');
+
+const headerTemplate = 'id,name,description,articleNumber,details,materials,sustainability,width,height,length,weight,packages,shortDesc,thread-count,Pillowcase quantity, Duvet cover length, Duvet cover width, Pillowcase length, Pillowcase width, fitting, imagesUrls\n'
+const writeRecords = fs.createWriteStream('database/records.csv');
+writeRecords.write(headerTemplate, 'utf8');
```

Comment:
Manually writing the headers at the start is a good idea so you don't forget. I am pretty sure that I forgot (my file is too big to view to check...) and will have to add them somehow. I will have to figure out a way to append to the beginning of a file, because I would rather not run my entire generation script again.